cmd/soroban-cli/src/tx/builder/bytesm.rs

```diff
@@ -0,0 +1,28 @@
+use std::str::FromStr;
+
+use crate::xdr;
+
+#[derive(Clone, Debug)]
+pub struct Bytes64(pub xdr::BytesM<64>);
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    Xdr(#[from] xdr::Error),
+    #[error(transparent)]
+    Hex(#[from] hex::FromHexError),
+}
+
+impl FromStr for Bytes64 {
+    type Err = Error;
+
+    fn from_str(value: &str) -> Result<Self, Self::Err> {
+        Ok(Self(xdr::BytesM::<64>::try_from(&hex::decode(value)?)?))
```

Comment:
The `xdr::BytesM` type should already support from_str of hex as we implement in stellar-xdr. What was the reason for needing to add that here?

Reply:
Good point! Removed!